schema_salad/makedoc.py

```diff
@@ -227,7 +234,7 @@ def typefmt(self,
                 nbsp=False,             # type: bool
                 jsonldPredicate=None    # type: Optional[Dict[str, str]]
                 ):
-        # type: (...) -> Union[str, unicode]
+        # type: (...) -> Union[str, Text]
```

Comment:
Isn't Text an alias for str in python3 and unicode in python2, why mention str separately?